client/state/reader/lists/test/actions.js

```diff
@@ -74,4 +102,46 @@ describe( 'actions', () => {
 			} );
 		} );
 	} );
+
+	describe( '#followList()', () => {
+		before( () => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.persist()
```

Comment:
don't need all the persists. Probably should use `useNock` from helpers as we chatted about at the meetup.